src/Menu.tsx

```diff
@@ -107,6 +107,7 @@ export interface MenuProps
   forceSubMenuRender?: boolean;
   triggerSubMenuAction?: TriggerSubMenuAction;
   builtinPlacements?: BuiltinPlacements;
+  overflowPopupOffset?: number[]
```

Comment:
From a functional and naming standard, should this be provided? @MadCcc

Reply:
i refer from this
https://github.com/react-component/menu/blob/4ae012f8276b9ac41182e030980d7dbc65ae3563/src/interface.ts#L30